src/ontology/mondo-edit.obo

```diff
@@ -46382,6 +46450,7 @@ def: "A metabolic disorder characterized by abnormal blood glucose levels." [NCI
 synonym: "disorder of glucose metabolism" EXACT [NCIT:C53655]
 synonym: "glucose metabolism disorder" EXACT [NCIT:C53655]
 xref: DOID:4194 {source="MONDO:equivalentTo"}
+xref: ICD10CM:E15-E16 {source="MONDO:subClassOf", source="https://orcid.org/0000-0001-5208-3432", source="https://orcid.org/0000-0002-4142-7153"}
```

Comment:
We say here that "disorder of glucose metabolism" is a subClassOf "Other disorders of glucose regulation and pancreatic internal secretion". The key being "other here" - are we certain that is what we want to do?

Reply:
this was a particularly confusing case, so no, I am not certain!

Reply:
I think I am neither.. We can discuss this on Wednesday!